daq/faucet_states_collector.py

```diff
@@ -60,9 +61,11 @@ def __init__(self):
         self.lock = Lock()
         self.learned_macs = self.system_states[KEY_LEARNED_MACS]
 
-    def get_system(self):
-        """get the system states"""
-        return self.system_states
+    def get_system_stats(self):
```

Comment:
_state

Reply:
Removed line since this will in another high level class